lib/src/component/metrics/threads-modal.js

```diff
@@ -90,8 +90,8 @@ var ThreadsModal = /** @class */ (function (_super) {
             counters = this.computeCounters();
             filteredList = this.computeFilteredList();
         }
-        return (React.createElement(reactstrap_1.Modal, { isOpen: showModal, toggle: handleClose, className: "modal-lg" },
-            React.createElement(reactstrap_1.ModalHeader, { toggle: handleClose }, "Threads dump"),
+        return (React.createElement(reactstrap_1.Modal, { isOpen: showModal, toggle: function () { return handleClose(); }, className: "modal-lg" },
+            React.createElement(reactstrap_1.ModalHeader, { toggle: function () { return handleClose(); } }, "Threads dump"),
```

Comment:
Why do we have to do this?

Reply:
The issue is in the tsx file about compilation.
I guess React 17 has changed somehow the signature of events... (?)
It kind of requires an event as parameter, that can be "ignored" by providing `() => handleClose()` (maybe I should do `(e) => handleClose(e)` instead ?)

The signature looks like this (I don't understand much of this): 
`type EventHandler<E extends SyntheticEvent<any>> = { bivarianceHack(event: E): void }["bivarianceHack"];`

Reply:
yes, its because of stricter type check. The right solution would be to match the signature by changing the original function. Adding this extra closure is not ideal. Can we change the `handleClose` function to take event and just ignore it?

Reply:
sure, this way ?